ale_linters/php/psalm.vim

```diff
@@ -0,0 +1,38 @@
+" Author: richard marmorstein <https://github.com/twitchard>
+" Description: plugin for Psalm, static analyzer for PHP
+
+let g:ale_php_psalm_executable = get(g:, 'ale_php_psalm_executable', 'psalm')
+
+function! ale_linters#php#psalm#GetExecutable(buffer) abort
+    return ale#Var(a:buffer, 'php_psalm_executable')
```

Comment:
You can write `'executable_callback': ale#VarFunc('php_psalm_executable')` below instead.